content/videos/challenges/71-minesweeper/index.json

```diff
@@ -0,0 +1,85 @@
+{
+  "title": "Minesweeper",
+  "description": "In this challenge, I attempt to code the classic game Minesweeper in JavaScript with the p5.js library.",
+  "videoNumber": "71",
+  "videoId": "LFU5ZlrR21E",
+  "date": "2017-05-18",
+  "languages": ["JavaScript", "p5.js"],
+  "topics": ["Games", "Prototypes", "Floodfill"],
+  "canContribute": true,
+  "timestamps": [
+    { "time": "0:00", "title": "Welcome!" },
+    { "time": "1:26", "title": "All aboard!" },
+    { "time": "2:36", "title": "Explaining the logic of the game" },
+    { "time": "4:31", "title": "Defining the cells" },
+    {
+      "time": "11:03",
+      "title": "Let's make a show() function using object protypes!"
+    },
+    { "time": "17:30", "title": "Checking if mouse is inside a grid" },
+    { "time": "20:15", "title": "Revealing the cells after clicking" },
+    {
+      "time": "22:58",
+      "title": "Checking how many bees are neighboring a particular cell"
+    },
+    { "time": "34:20", "title": "Starting to work on the game mechanic!" },
+    { "time": "41:50", "title": "Let's implement floodfill!" },
+    { "time": "48:45", "title": "Defining when the game is over" },
+    { "time": "51:11", "title": "Minesweeper!" },
+    { "time": "53:12", "title": "Thanks for watching!" }
+  ],
+  "codeExamples": [
+    {
+      "title": "Minesweeper",
+      "description": "A simple Minesweeper game",
+      "image": "minesweeper.png",
+      "urls": {
+        "p5": "https://editor.p5js.org/codingtrain/sketches/-mz-ePoqd",
+        "processing": "https://github.com/CodingTrain/Coding-Challenges/tree/main/071_minesweeper/Processing",
+        "other": "https://github.com/CodingTrain/Coding-Challenges/tree/main/071_minesweeper"
```

Comment:
Noting we can skip the "other" property if it's just the same as the Processing code. Other is really just for if it's not p5 or Processing.